schema/partner_show.js

```diff
@@ -269,7 +278,7 @@ const PartnerShow = {
   resolve: (root, { id }) => {
     return gravity(`show/${id}`)
       .then(show => {
-        if (!show.displayable) return new Error('Show Not Found');
+        if (!show.displayable && !show.is_reference) return new Error('Show Not Found');
```

Comment:
We sort of use `displayable` to mean something ambiguous. It essentially means if the show has a standalone page on the site, which ref shows do not. I'm not sure if this was the best place to put this `Error` checking. I think the standalone show page on the site (powered by MP) should do this check, since it's non-standard.